ote_sdk/ote_sdk/utils/segmentation_utils.py

```diff
@@ -142,6 +143,51 @@ def create_hard_prediction_from_soft_prediction(
     return hard_prediction
 
 
+Contour = List[Tuple[float, float]]
+
+
+def get_subcontours(contour: Contour) -> List[Contour]:
+    """
+    Splits contour into subcontours that do not have self intersections.
+    """
+
+    ContourInternal = List[Optional[Tuple[float, float]]]
```

Comment:
Why do you need to convert it to float and then compare instead of comparing int values?

Reply:
This is just type annotation. It has no effect on the commands that are executed at runtime, i.e. no conversion is happening. It just says mypy then float and its subclasses are acceptable here. And `int` is a subclass of `float`. [PEP-484](https://www.python.org/dev/peps/pep-0484/#the-numeric-tower)